ctk_cli/module.py

```diff
@@ -230,7 +232,7 @@ def identifier(self):
 
     def parseValue(self, value):
         """Parse the given value and return result."""
-        if self.isNumericVector():
+        if self.isNumericVector() or self.typ == 'string-vector':
```

Comment:
Should we instead add a `isVector` function that would return `True` for all vector type ?